frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -254,8 +239,33 @@ private void processSuccessfulCodeRequest(
         }
         codeStorageService.deleteOtpCode(session.getEmailAddress(), notificationType);
 
+        var metadataPairArray =
+                metadataPairs(notificationType, journeyType, codeRequest, loginFailureCount, true);
         auditService.submitAuditEvent(
-                FrontendAuditableEvent.CODE_VERIFIED, auditContext, metadataPairs);
+                FrontendAuditableEvent.CODE_VERIFIED, auditContext, metadataPairArray);
+    }
+
+    private AuditService.MetadataPair[] metadataPairs(
+            NotificationType notificationType,
+            JourneyType journeyType,
+            VerifyCodeRequest codeRequest,
+            Integer loginFailureCount,
+            Boolean successfulRequest) {
+        AuditService.MetadataPair[] basicMetadataPairs = {
+            pair("notification-type", notificationType.name()),
+            pair("account-recovery", journeyType.equals(JourneyType.ACCOUNT_RECOVERY)),
+            pair("journey-type", String.valueOf(journeyType))
+        };
+        List<AuditService.MetadataPair> elementsList = new ArrayList<>(List.of(basicMetadataPairs));
```

Comment:
We're starting with an array, then creating a list, then converting back to an array in the final line.

Could we start with just a list instead? e.g. `var elementsList = Arrays.asList(pair("notification-type", ...)...)`